### ISSUE TITLE
Cannot dataprep images, structured by FiJi macros.

### ISSUE THREAD

**Issue by dimethylglymaxim** [SchmollerLab/Cell_ACDC#212]
### Description:

As far as I understand, FiJi macros (for multiple files) do not create a metadata file and therefore cannot be opened by the data prep module. 

### Way to reproduce:

1. Create data structure using provided macros (for multiple files, on multiple time-lapse movies).
2. Run CellACDC data prep module.
3. Load positions from the created data structure.
4. Fail miserably.

### Terminal output:

```Shell
Initializing Data Prep module v1.2.4rc53...
Loading Y:/lab/Max/EXP3/Segmentation\Position_1\Images/xy01_s01_gray.tif...
Traceback (most recent call last):
  File "E:\ACDC\envs\acdc\lib\site-packages\cellacdc\dataPrep.py", line 2164, in openFolder
    self.loadFiles(exp_path, user_ch_file_paths, user_ch_name)
  File "E:\ACDC\envs\acdc\lib\site-packages\cellacdc\dataPrep.py", line 2017, in loadFiles
    proceed = self.init_data(user_ch_file_paths, user_ch_name)
  File "E:\ACDC\envs\acdc\lib\site-packages\cellacdc\dataPrep.py", line 1195, in init_data
    proceed = posData.askInputMetadata(
  File "E:\ACDC\envs\acdc\lib\site-packages\cellacdc\load.py", line 1352, in askInputMetadata
    if SizeT_metadata>1 and numPos>1 and warnMultiPos:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

**Comment by ElpadoCan** [comment]
Hi @dimethylglymaxim 

Sorry, I haven't used the Fiji Macros for months, so I never had a Position without the metadata file. 

In a recent update, we added a warning when you are trying to load multiple timelapse positions into the main GUI. However, without the metadata, it should not try to check this because it cannot know whether it is dealing with timelapse or not. 

Therefore, I just released `v1.2.4rc54` which should fix this. To update, please activate the `acdc` environment and then run the command `pip install --upgrade cellacdc`. Let me know if it works, thanks! 

Cheers,
Francesco

PS: DId you try using the module `0. Create data structure from microscopy/image file(s)...`? It should be way more robust than the Fiji macros. It's not mandatory to use anyway, so thanks for reporting this issue that will certainly help other users using the Fiji macros.

**Comment by dimethylglymaxim** [comment]
Yep, it works now. Thank you very much.
